### ISSUE TITLE
Use aspects in packages instead of pragmas

### ISSUE THREAD

**Issue by jklmnn** [Componolit/ada-style#6]
In subprograms we already use aspects as much as possible instead of pragmas. In packages however we mostly still use pragmas. I'd propose to use aspects here, too:
```Ada
package P with
   SPARK_Mode,
   Pure
is
end P;
```
instead of
```Ada
package P with
   SPARK_Mode
is
   pragma Pure;
end P;
```
In some cases this might even prevent bug boxes. I once had a bugbox when using `pragma Pure` that I could fix when using the aspect instead. Unfortunately there's no workaround entry and I don't have a reproducer anymore.

**Comment by senier** [comment]
I think I remember a case where it was the other way round - using a pragma instead of an aspect avoided a bug. However, I agree that this should be the only excuse to use a pragma over an aspect (and this pragma should then be annotated by a Workaround).

**Comment by jklmnn** [comment]
Fixed in #8.
